src/Features/CSharp/Portable/ExtractMethod/CSharpSelectionResult.cs

```diff
@@ -95,6 +97,37 @@ protected override bool UnderAsyncAnonymousMethod(SyntaxToken token, SyntaxToken
                    current.GetLastToken().Span.End <= lastToken.Span.End;
         }
 
+        protected sealed override bool IsConfigureAwaitFalse(SyntaxNode node)
+        {
+            if (node is InvocationExpressionSyntax
+            {
+                Expression: MemberAccessExpressionSyntax
+                {
+                    Name:
+                    {
+                        Identifier:
+                        {
+                            ValueText: "ConfigureAwait"
+                        }
+                    }
```

Comment:
consider: `Name: { Identifier: { ValueText: nameof(ConfigureAwait) } }`.    This reads fine to me as an atomic description of hte name.

Reply:
Again VS only allows patterns to be all one line, or every brace a new line.

Reply:
i say this honestly :)   I would gate the usae of these patterns on us fixing that that behavior of the formatter :)   I don't think it would be that hard.  I think we basically only need to add the "SuppressOperation" on htese pattersn if they're on a single line.

Reply:
I'm not sure I have the courage to face the formatter :-)

If I do fix it, how long would it take to pipe through though? Till the release of VS, or does Roslyn get it sooner?

Reply:
yeah, we shouldn't gate on this.  but can you file an issue on this at least?  

It would take a while to pipe through.  but, in the meantime, you can just have code that doesn't match teh formatter.  we had that for a while with switch-expressions.

Reply:
it just goes with hte territory with adopting new lang features before the tooling is totally up to date with it :)

Reply:
> I'm not sure I have the courage to face the formatter :-)

In this case i actually think it would be simple.  I can try to walk you through it if you want!